numpy/random/tests/test_random.py

```diff
@@ -205,12 +208,13 @@ def test_repeatability(self):
         assert_(tgt[np.dtype(np.bool).name] == res)
 
 
-class TestRandomDist(TestCase):
+class TestRandomDist:
```

Comment:
This will be an old style class on Python 2.7, best derive from object if not `TestCase`. Any reason to avoid `TestCase`?